functions/src/index.ts

```diff
@@ -21,7 +21,33 @@ main.use(bodyParser.json());
 export const getCVEFunction = CVEFunction.getCVE;
 export const getSPLFunction = SPLFunction.getSPL;
 export const getBulletinFunction = bulletinFunction.getBulletin;
-export const getAndroidVersionFunction = androidVersionFunction.getAndroidVersion;
-export const storeEmailFunction = notificationFunction.accountCreate;
+export const getAndroidVersionFunction =
+  androidVersionFunction.getAndroidVersion;export const storeEmailFunction = notificationFunction.accountCreate;
 export const notifyNewVersionFunction = notificationFunction.notifyNewVersion;
 export const notifyNewReleaseFunction = notificationFunction.notifyNewRelease;
+
+export const getData = functions.https.onRequest(main);
+
+app.post('/data', (request: any, response: any) => {
+  if (request.body['email']) {
```

Comment:
You don't have to pass email to this function. If you implement the authentication function from https://github.com/firebase/functions-samples/blob/master/authorized-https-endpoint/functions/index.js
the email will be in the req.user object (from the decoded token).